fecgov/fec-cms
# Calculate presidential bulk data coverage dates

## Summary (required)

- Resolves #3626
- Calculate presidential bulk data coverage dates from lookup 22ce1d8579df67d3039f50ea425eccaa747dac90
- Remove unused `load_presidential_map_candidate_data` 12dcd06418012d74c251d45c623b037818f60066

## Impacted areas of the application

List general components of the application that this PR will affect:

-  https://www.fec.gov/data/browse-data/?tab=candidates

## Screenshots

No visual change, this part of the page is now dynamic: 
```
<p>The 2020 files contain financial activity through {{ monthly_coverage_date }} for presidential monthly filers and through {{ quarterly_coverage_date }} for presidential quarterly filers.</p>
```

<img width="706" alt="Screen Shot 2020-03-17 at 3 34 34 PM" src="https://user-images.githubusercontent.com/31420082/76895369-3e560880-6866-11ea-9501-6c0fa493d1ee.png">


## Related PRs

https://github.com/fecgov/fec-cms/pull/3622 "new presidential links on browse data/candidates page"

## How to test

- Turn on env var `export FEC_FEATURE_PRESIDENTIAL_MAP=True`
- Go to http://localhost:8000/data/browse-data/?tab=candidates
- Dates  should be correct according to https://docs.google.com/spreadsheets/d/1PQ4iTWHpN4Q1eTy-FCKBbSZSEMBcSfHRcfwRPjMtuGQ/edit#gid=0
<img width="706" alt="Screen Shot 2020-03-17 at 3 34 34 PM" src="https://user-images.githubusercontent.com/31420082/76895377-431abc80-6866-11ea-80ac-cdc607a9cc7a.png">

- Run `pytest`
- Test how it will act next Saturday (3/21) by having `utils.get_tomorrow()` return `datetime.datetime(2020, 3, 22)`
